.npmrc

```diff
@@ -0,0 +1 @@
+package-lock=false
```

Comment:
Meh, we could delete this IMO (and just add `yarn.lock` and `package-lock.json` to the `.gitignore` file).